### PR TITLE
[Task][27213] Create layout

### PR DESCRIPTION
**Pull request by nguyenvd16**
https://edu-redmine.sun-asterisk.vn/issues/27213

### CODE REVIEW

**Review by kguen-sun — APPROVED** [review]

**Review by nthtra-110273 — APPROVED** [review]

File: app/views/layouts/_sidebar.html.erb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    </div>
+
+    <!-- Sidebar Menu -->
+    <nav class="mt-2">
+      <ul class="nav nav-pills nav-sidebar flex-column" data-widget="treeview" role="menu" data-accordion="false">
+        <li class="nav-item has-treeview">
+          <a href="#" class="nav-link">
+            <i class="nav-icon fas fa-tachometer-alt"></i>
+            <p>
+              Dashboard
```
Comment (kguen-sun) [inline app/views/layouts/_sidebar.html.erb]: i18n anh ơi

File: app/views/layouts/_sidebar.html.erb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+              <i class="right fas fa-angle-left"></i>
+            </p>
+          </a>
+          <ul class="nav nav-treeview">
+            <li class="nav-item">
+              <a href="../../index.html" class="nav-link">
+                <i class="far fa-circle nav-icon"></i>
+                <p>Dashboard v1</p>
```
Comment (kguen-sun) [inline app/views/layouts/_sidebar.html.erb]: i18n

File: app/views/layouts/_sidebar.html.erb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              </a>
+            </li>
+          </ul>
+        </li>
+        <li class="nav-item">
+          <a href="../widgets.html" class="nav-link">
+            <i class="nav-icon fas fa-th"></i>
+            <p>
+              Widgets
```
Comment (kguen-sun) [inline app/views/layouts/_sidebar.html.erb]: i18n

File: app/views/layouts/_sidebar.html.erb [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+              <span class="right badge badge-danger">New</span>
+            </p>
+          </a>
+        </li>
+
+        <li class="nav-item">
+          <a href="../widgets.html" class="nav-link">
+            <i class="nav-icon fas fa-chart-pie"></i>
+            <p>
+              Charts
```
Comment (kguen-sun) [inline app/views/layouts/_sidebar.html.erb]: i18n

File: app/views/layouts/_sidebar.html.erb [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            </p>
+          </a>
+        </li>
+
+        <li class="nav-item">
+          <a href="../widgets.html" class="nav-link">
+            <i class="nav-icon fas fa-th"></i>
+            <p>
+              Widgets
```
Comment (kguen-sun) [inline app/views/layouts/_sidebar.html.erb]: i18n

**Comment by nguyenvd16** [comment]
ready

**Review by nthtra-110273 — APPROVED** [review]

**Review by kguen-sun — APPROVED** [review]

**Review by kguen-sun — APPROVED** [review]

**Review by nthtra-110273 — APPROVED** [review]

File: app/views/layouts/_sidebar.html.erb (hunk 1 above)
Comment (nguyenvd16) [reply app/views/layouts/_sidebar.html.erb]: done

**Pull request merged**